Khan/wonder-blocks
# Remove display value from Link

## Summary:
Currently, Links have `display: inline-flex` in their styles.
This was added as part of the recent Link updates, but it is
unclear as to why.

This caused some course names to be cut off without ellipses.
(See https://khanacademy.slack.com/archives/C04E2SED13J/p1681138678986699)

After doing some experimentation, it seems like the `display`
style doesn't even need to be set for Links by default. So here,
I am removing it.

Issue: none

## Test plan:
- Go to http://localhost:51027/?path=/story/link--variants.
- Resize the window.
- Confirm that all the links text wrap when they overflow.

## Before (inline-flex): 

https://user-images.githubusercontent.com/13231763/231018582-60225e45-2b82-41c9-8a95-a69faadf21f6.mov

## After (no display, default to inline in storybook):

https://user-images.githubusercontent.com/13231763/231018701-240b6228-cc5f-4950-996f-31b1f5893ac2.mov

### Note the outline on text wrap

<img width="740" alt="Screen Shot 2023-04-10 at 4 37 19 PM" src="https://user-images.githubusercontent.com/13231763/231019387-413682a4-19a9-4171-91a5-f551069ab2ca.png">
